pkg/plugin/oauth2/setup.go

```diff
@@ -152,7 +158,42 @@ func setupOAuth2(def *proxy.RouterDefinition, rawConfig plugin.Config) error {
 		return err
 	}
 
-	def.AddMiddleware(NewKeyExistsMiddleware(manager))
+	var kafkaConfig config.Config
+
+	_ = config.UnmarshalYAML("./config/config.yaml", &kafkaConfig)
+	if kafkaConfig.KafkaAddr == "" {
+		fmt.Errorf("Kafka adress not setted!")
```

Comment:
is it fatal?

Reply:
this is a strange error, there is a config file in the docker container, i can read it but it always drops here.